src/libs/Navigation/Navigation.js

```diff
@@ -54,6 +55,7 @@ function goBack(shouldOpenDrawer = true) {
  * @param {String} route
  */
 function navigate(route = ROUTES.HOME) {
+    Log.info('Navigating to route', false, {route});
```

Comment:
NAB. Nice addition of this log! It might be even better to add a logger when the navigation state changes. Which will also capture things like `goBack()` or `openDrawer()`. Probably can be done later as 90% of navigations use this method - but thought you might like to know [this](https://reactnavigation.org/docs/navigation-container/#onstatechange) exists.

Reply:
So, I would need to add the log in [this](https://github.com/Expensify/App/blob/01e0e29eff7a5b6baad00b37c9d674f6ef2fedfa/src/libs/Navigation/NavigationRoot.js#L39) event right (and remove the logging from here)?

Reply:
Yep, I think the "currentURL" or "path" is probably the most valuable to have now and moving the log there should capture everything and not just when we explicit use of `navigate()`?

There's also some other information in the "state" besides the path that could be useful to log - but less sure about that - more info here if you are curious.

https://reactnavigation.org/docs/navigation-state/